<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sreekumar" name="Default test" tests="4" failures="1" timestamp="17 Mar 2021 06:56:10 GMT" time="52.656" errors="0">
  <testcase name="lufthansaGotoLoginPageTest" time="0.008" classname="com.lufthansa.qa.testcases.LufthansaHomePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.lufthansa.qa.pages.LufthansaHomePage.goToLoginPage(LufthansaHomePage.java:53)
	at com.lufthansa.qa.testcases.LufthansaHomePageTest.lufthansaGotoLoginPageTest(LufthansaHomePageTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- lufthansaGotoLoginPageTest -->
  <testcase name="lufthansaHomePageTitleTest" time="0.096" classname="com.lufthansa.qa.testcases.LufthansaHomePageTest"/>
  <testcase name="lufthansaLoginButtonValidationTest" time="0.184" classname="com.lufthansa.qa.testcases.LufthansaHomePageTest"/>
  <testcase name="lufthansaLogoTest" time="0.227" classname="com.lufthansa.qa.testcases.LufthansaHomePageTest"/>
</testsuite> <!-- Default test -->
